.github/workflows/main.yml

```
@@ -79,6 +80,33 @@ jobs:
           path: ./*
           key: ${{ github.ref }}-${{ github.sha }}-build-release
 
+  build-docs:
+    needs: setup
+    runs-on: ubuntu-latest
+    steps:
+      - name: Restore setup
+        uses: actions/cache@v3
+        with:
+          path: ./*
+          key: ${{ github.ref }}-${{ github.sha }}-setup
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ env.RUBY_VERSION }}
+          bundler-cache: true
```

We should add the `working-directory` here as well.
```suggestion
          bundler-cache: true
          working-directory: ./docs
```

---

Done 🙏